src/core/jwk.rs

```diff
@@ -49,6 +50,31 @@ pub struct CoreJsonWebKey {
     )]
     pub(crate) e: Option<Base64UrlEncodedBytes>,
 
+    //Elliptic Curve
+    #[serde(
+        skip_serializing_if = "Option::is_none"
```

Comment:
let's add `default` and `deserialize_with = "deserialize_option_or_none"` here too, I think. this struct currently uses `deserialize_option_or_none` for all of the cipher-specific fields for the reason described on lines 36-39.